cosmos/dbt/graph.py

```diff
@@ -348,7 +378,14 @@ def run_dbt_ls(
         self, dbt_cmd: str, project_path: Path, tmp_dir: Path, env_vars: dict[str, str]
     ) -> dict[str, DbtNode]:
         """Runs dbt ls command and returns the parsed nodes."""
-        ls_command = [dbt_cmd, "ls", "--output", "json"]
+        ls_command = [
+            dbt_cmd,
+            "ls",
+            "--output",
+            "json",
+            "--output-keys",
+            '"name alias unique_id resource_type depends_on original_file_path tags config freshness"',
```

Comment:
Why is the `--output-keys` option necessary? I also suspect that the CI failure might be connected to this change.

Reply:
yes, dbt ls doesn't output freshness by default.

I'm having a hard time debugging this, since the output in the tests is {}
but I'm testing locally with an env running dbt-core=1.5.4 and dbt-postgres=1.5.4
and the command does work from my terminal

Reply:
Could you please test once with 
```
ls_command = [
 dbt_cmd,
 "ls",
 "--output",
 "json",
 "--output-keys",
 "name",
 "unique_id",
 "resource_type",
 "depends_on",
 "original_file_path",
 "tags",
 "config",
 "freshness"
 ]
```

Reply:
That worked! thank you

Reply:
yes, But look like now failing for dbt-1.4